{"uuid":"566dfbf9-448c-4bdb-9ac1-8838630863c6","name":"Scenario Outline: The user is presented with error message for code with invalid syntax in tryEditor for Arrays in Python page","children":["a1d10ada-cdf8-4cbc-80b9-2aa3cec2673b"],"befores":[{"name":"appHooks.Hooks.scenario(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673380995975,"stop":1673380995975}],"afters":[{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673380995982,"stop":1673380995982},{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673380996255,"stop":1673380996256},{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673380996439,"stop":1673380996439},{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[{"name":"My screenshot","source":"02f3eec7-b34b-4e5e-8ef4-63456bde2fe6-attachment","type":"image/png"},{"name":"Myscreenshot","source":"d7a9497a-2c2e-433e-83a0-8e7afb8907b3-attachment"}],"parameters":[],"start":1673380996450,"stop":1673380996733}],"start":1673380995975,"stop":1673380996733}